src/components/layout/modal/update_browser_alert_modal.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react'
+import { ModalRefType } from 'components/elements/modal'
+import { Localize } from 'components/localization'
+import { Modal, Header } from 'components/elements'
+import { useUserBrowser } from 'components/hooks/use-user-browser'
+import { CookieStorage } from 'common/storage'
+
+const BrowserUpdateAlertModal = () => {
```

Comment:
Can we rename the file to , [browser_update_alert_modal.tsx] ?

Reply:
yes. will change it